apps/server/src/modules/task/mapper/task.mapper.ts

```diff
@@ -23,16 +24,8 @@ export class TaskMapper {
 
 		dto.displayColor = taskDesc.color;
 		dto.description = taskDesc.lessonName;
+		dto.lessonHidden = taskDesc.lessonHidden;
 
 		return dto;
 	}
```

Comment:
why was this method deleted?

Reply:
by accident - but saw the errors this morning, too. Thanks for checking it, too.

Reply:
we'll have some conflicts :)
https://github.com/hpi-schul-cloud/schulcloud-server/pull/3682